MM-30858. Added documentation about versions.

rachelbritton opened the pull request:
And updated several of the overhead pages.

rachelbritton committed:
MM-30858. Added documentation about versions.

And updated several of the overhead pages.

rachelbritton committed:
Merge branch '25.1' into issues/MM-30858-calendar-based-versioning

rachelbritton committed:
MM-30858. Adjusted text.

agudgeirsson commented on versions.rst:
```diff
@@ -0,0 +1,25 @@
+
+Micetro uses calendar-based versioning, in which releases are numbered according to the year in which they are released, along with two rolling numbers that indicate the exact release number within a calendar year. In this methodology, version numbers are structured as **Year.Release.Patch**. For example, in the version 25.1.1:
+
+* **Year** indicates the last two digits of the calendar year, e.g., 25 means the version was released in the year 2025. The first two digits of the year (e.g., 20) are not used.
```
I think we can skip `The first two digits of the year (e.g., 20) are not used.` It think that clear from the first part of the sentence

agudgeirsson commented on versions.rst:
```diff
@@ -0,0 +1,25 @@
+
+Calendar-based versioning will not be applied retroactively to versions that were released before this change, i.e., 11.1.0 will remain 11.1.0.
+
+Each release builds on top of the previous releases, so any new release contains fixes for bugs and issues that were resolved in previous releases. Patches are only used when specific issues in a particular release need to be addressed. Any fixes applied in a patch will also be aprt of the next release and any releases in the future.
```
part (not aprt)

agudgeirsson commented on versions.rst:
```diff
@@ -0,0 +1,25 @@
+Calendar-based versioning will not be applied retroactively to versions that were released before this change, i.e., 11.1.0 will remain 11.1.0.
+
+Each release builds on top of the previous releases, so any new release contains fixes for bugs and issues that were resolved in previous releases. Patches are only used when specific issues in a particular release need to be addressed. Any fixes applied in a patch will also be aprt of the next release and any releases in the future.
+Releases are supported for two years from their original release dates. Any patches applied on top of a release do not reset this timeframe, which always refers to the release date for which .0 is the patch number.
```
Extra space before this line ?

rachelbritton committed:
MM-30858. Revised text.

rachelbritton committed:
MM-30858. Adjusted text based on feedback.

Adjusted text to align with language used in comms about the versioning change and with Valli's feedback.

Will link to this page from the callout in the release notes.

valgeir commented on versions.rst:
```diff
@@ -0,0 +1,30 @@
+Versions
+========
+
+Starting with version 25.1.0, Micetro uses a calendar-based versioning model, in which releases are numbered according to the year in which they are released, along with two rolling numbers that indicate the exact release number within a calendar year. In this versioning model, version numbers are structured as **Year.Release.Patch**. For example, in the version 25.1.1:
```
I got a little confused when we introduce the two rolling numbers, which I like, but then explain their purpose together.

Since we say exact release version but later say the middle integer is for release and third integer for patch,

Maybe its just be but I think it could be a little clearer to explain the two rolling integers separately

valgeir commented on versions.rst:
```diff
@@ -0,0 +1,30 @@
+
+Calendar-based versioning will not be applied retroactively to existing versions, e.g., 11.1.0.
+
+Each release builds on top of the previous releases, so any new release contains fixes for bugs and issues that were resolved in previous releases. Patches are only used when specific issues in a particular release need to be addressed. Any fixes applied in a patch will also be part of the next release and any releases in the future.
```
Very good!

valgeir commented on versions.rst:
```diff
@@ -0,0 +1,30 @@
+----------------------------------------
+If you're using a version of Micetro that is not the most recent, you can access the version-specific documentation---including an installation guide, user guide, admin guide, and reference articles---by selecting your version from the dropdown in the upper left corner of the screen.
+
+Because year releases are supported for two years, the documentation for year releases are also maintained for two years.
```
We might need a better term than year releases

rachelbritton committed:
MM-30858. Updated version change info.

Updated the version change documentation to cohere with how the company is talking about it, e.g., other BlueCat product documentation.

rachelbritton committed:
MM-30858. Removed old link.

Removed link to deprecated content.

rachelbritton committed:
MM-30858. Amended legal text and link.

rachelbritton committed:
Fixed typo.
